### PR TITLE
Create Collections page

### PR DESCRIPTION
**Pull request by miguelrodrigues22**
Create page for collections

Also (home page related):
  - Create navbar component (without notification list)
  - Create contests card
  - Add filter by All, Open, Voting and Closed
  - Created contest filter reusable component, to be used, both in the home page and the contests page.

### CODE REVIEW

File: components/ContestFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import SearchInput from './SearchInput'
+
+function ContestFilter(props) {
+  const { open, setOpen, statusFilter, setStatusFilter } = props
+
+  const handleStatusFilter = (status: string) => {
+    setStatusFilter(status)
+  }
+
+  const selectedFilterCSS = (filter) => {
```
Comment (JdFSilva) [inline components/ContestFilter.tsx:11]: add the filter interface as the type of the filter

File: components/ContestFilter.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      {open && (
+        <div>
+          <button
+            className={`${selectedFilterCSS(
+              'All'
+            )} text-white px-3 py-2 rounded-2xl font-medium cursor-pointer mr-2 mt-2`}
+            onClick={() => handleStatusFilter('All')}
+          >
+            All
+          </button>
```
Comment (JdFSilva) [inline components/ContestFilter.tsx:39]: the filter buttons can be created with a for loop that iterates over a list of the possible filters ["All", "Open", "Voting", "Closed"]

File: components/Contest.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+interface Contest {
+  name: string
+  description: string
+  date_end: string
+  status: string
```
Comment (JdFSilva) [inline components/Contest.tsx:7]: the status should be the same interface as the filters, not a general string

File: components/Collection.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          <p className='text-gray-700 text-base'>{collection.user}</p>
+
+          {/* {collection.pictures?.map((picture) => (
+            <Image
+              src={picture.picture_path}
+              alt='Cleek'
+              width={40}
+              height={40}
+            />
+          ))} */}
```
Comment (JdFSilva) [inline components/Collection.tsx:26]: you can remove this

File: components/Collection.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import Image from 'next/image'
+
+export default function Collection(props) {
+  const { collection } = props
+  return (
+    <>
+      <div className='max-w-sm rounded overflow-hidden shadow-lg'>
+        <img
+          className='w-full'
+          src='https://www.thoughtco.com/thmb/LIkTVpCr3NAa7qJW7D57BXgRiJA=/3396x2547/smart/filters:no_upscale()/film-photography-592347645-59e4d0609abed500119e7b14.jpg'
```
Comment (JdFSilva) [inline components/Collection.tsx:11]: save this to public/images

File: components/Collection.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Image from 'next/image'
+
+export default function Collection(props) {
+  const { collection } = props
```
Comment (JdFSilva) [inline components/Collection.tsx:5]: create an interface for the props
```
export default function Collection({ collection }: CollectionProps) {
```

File: components/Navbar/Navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+                >
+                  Photo Contests
+                </a>
+                <a
+                  onClick={() => handleNavigation('/collections')}
+                  className={
+                    router.pathname === '/collections'
+                      ? `${collectionsTextColor} hover:bg-orange-700 hover:text-white px-3 py-2 rounded-md font-medium cursor-pointer`
+                      : `${collectionsTextColor} hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md font-medium cursor-pointer`
+                  }
```
Comment (JdFSilva) [inline components/Navbar/Navbar.tsx:51]: same as above

**Review by JdFSilva — DISMISSED** [review]

File: components/Contest.tsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (miguelrodrigues22) [reply components/Contest.tsx:1]: it does not, will look into it

**Review by tiagocardoso98 — APPROVED** [review]

**Pull request merged**